src/components/CustomButton.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { CustomButtonProps } from '../../types'
+const CustomButton = ({title,active,disabled, handleClick}: CustomButtonProps) => {
+  return (
+    <button  onClick={handleClick} className='bg-[#111111] px-16 py-4 border-2 border-[#ffcb74] rounded-full text-[#ffcb74] text-2xl font-bold uppercase'>
+           {title}
+
+    </button>
```

Comment:
Please make sure to always format your code neatly
Line 3 give spaces to allow your props to breath